modules/doc/content/application_development/build_system.md

```diff
@@ -129,16 +142,22 @@ Make sure to prepend your directories with a `%` sign.
 
 ### Revision Generation
 
-The build process can generate C++ header files containing repository versioning information for use within your application. These header files are generated by default but can be disabled by setting a Make variable in your application's Makefile:
+!style! halign=left
+The build process can generate C++ header files containing repository versioning information for use
+within your application. These header files are generated by default but can be disabled by setting
+a Make variable in your application's Makefile:
+!style-end!
 
 ```
 # must be set before the app.mk file is included
 GEN_REVISION := no
 include      $(FRAMEWORK_DIR)/app.mk
 ```
 
-The Revision file contains both a version string (git hash) and a revision string containing more detailed information or tagged information. Note that the
-name of your app will appear instead of "MOOSE" for your individual application:
+The Revision file contains both a version string (git hash) and a revision string containing more
+detailed information or tagged information. Note that the name of your app will appear instead of
+"MOOSE" for your individual application:
+
```

Comment:
lol, this was never rendering properly.

ref: https://mooseframework.inl.gov/application_development/build_system.html#f257438f-88ba-4daf-887e-8e03cce37149

Reply:
Those empty lines are super easy to forget....